lib/utils/format.js

```diff
@@ -0,0 +1,14 @@
+export default (function() {
+  String.prototype.format = function () {
```

Comment:
Extending native prototypes makes my linter explode... I can either add an exception for "String" to the ESLint config, or we can make this an actual utility function where you'd call it on a string, whichever you prefer.